service/awsconfig/v7/templates/cloudformation/guest/outputs.go

```diff
@@ -8,6 +8,8 @@ Outputs:
     Value: {{ .Outputs.Master.InstanceType }}
   MasterCloudConfigVersion:
     Value: {{ .Outputs.Master.CloudConfig.Version }}
+  {{ .Outputs.Worker.ASG.Key }}:
```

Comment:
I thought it would be good to reuse the keys we use to access the actual values later on. Then we have them in one place and do not miss something when we change things in one place.